### DISCUSSION TITLE
Activity (Bug, i don't find the category)

### DISCUSSION THREAD

**Discussion by JoanFo1456** [pelican-dev/panel#567] [General]
Someone told me to open here a bug and i didn't see where to click, so i'm using this, i'm sorry.
When a user does something in activity, delete files, etc... And you delete the user, shows the activity as the system's one with the user IP, it's a little bit confusing.
Here is some proof.
![image](https://github.com/user-attachments/assets/115cf35c-4925-42d3-bd6a-19ce49d8ba6e)

**Comment by notAreYouScared** [comment]
Well, the user got deleted, and we don't store users in the database. They're just a number, therefore the system takes their place.

**Reply by Boy132** [reply]
We should show something like `Deleted User` instead of `System` nevertheless.

**Comment by Boy132** [comment]
See #569
